pkg/apis/core/validation/validation.go

```diff
@@ -3011,6 +3012,37 @@ func validatePullPolicy(policy core.PullPolicy, fldPath *field.Path) field.Error
 	return allErrors
 }
 
+var supportedResizeResources = sets.NewString(string(core.ResourceCPU), string(core.ResourceMemory))
+var supportedResizePolicies = sets.NewString(string(core.RestartNotRequired), string(core.RestartRequired))
+
+func validateResizePolicy(policyList []core.ContainerResizePolicy, fldPath *field.Path) field.ErrorList {
```

Comment:
@vinaykul 
I think this function is only called by the test. Is it what we expected?

Reply:
I don't mind.
Anyway, is that a release blocker? or we can fix it in the patch release?

Reply:
it's not ideal, and I'd still consider accepting a fix for 1.27 if at all possible, but IMO it's not a release blocker; validation of alpha data is allowed to change in incompatible ways between releases with release notes, the go field type just can't change

@vinaykul please add this to the list of known issues referenced in any user-facing documentation about the feature, since users trying out the feature can put incorrect values in the field that won't work properly

Reply:
opened a bug issue [here](https://github.com/kubernetes/kubernetes/issues/116854).